Library/Formula/mysql-cluster.rb

```diff
@@ -1,24 +1,25 @@
-require 'formula'
-
 class MysqlCluster < Formula
-  homepage 'http://www.mysql.com/cluster/'
-  url 'http://mysql.llarian.net/Downloads/MySQL-Cluster-7.3/mysql-cluster-gpl-7.3.4.tar.gz'
-  sha1 '01003903da6802885bc98565ebe07f76714488fe'
+  homepage "http://www.mysql.com/cluster/"
+  url "http://dev.mysql.com/get/Downloads/MySQL-Cluster-7.3/mysql-cluster-gpl-7.3.8.tar.gz"
+  sha1 "f70ac7955343765146c556576c8b13dbedf9c593"
+
+  # Fix me: if you can get this to build on Yosemite, send a pull request!
+  depends_on MaximumMacOSRequirement => :mavericks
```

Comment:
I'd like to suggest changing `homepage` to `https://www.mysql.com/products/cluster/` (to avoid a redirect and enable SSL) and `url` to https: `https://dev.mysql.com/get/Downloads/MySQL-Cluster-7.3/mysql-cluster-gpl-7.3.8.tar.gz`